cpp/src/barretenberg/proof_system/types/composer_type.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+namespace proof_system {
```

Comment:
Again, assuming this is to be addressed imminently and does not really require a TODO?